package-lock.json

```diff
@@ -3563,7 +3562,7 @@
     "fastparse": {
       "version": "1.1.2",
       "resolved": "https://registry.npmjs.org/fastparse/-/fastparse-1.1.2.tgz",
-      "integrity": "sha512-483XLLxTVIwWK3QTrMGRqUfUpoOs/0hbQrl2oz4J0pAcm3A3bu84wxTFqGqkJzewCLdME38xJLJAxBABfQT8sQ==",
+      "integrity": "sha1-kXKMWllC7O2FMSg8eUQe5BIsNak=",
```

Comment:
Undo these changes.

Reply:
It happened automatically, I will revert this.

Reply:
Done